test/getCoverageLIb.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { describe, it } from "node:test";
+import assert from "node:assert";
+import { getCoverageLib } from "../src/utils";
+
+describe("getCoverageLib.js", () => {
+  it("getCoverageLib should return N/A", () => {
```

Comment:
```suggestion
  it("should return N/A", () => {
```

You don't need to add the function name in each test because we already know it with the `describe`